src/components/TableWithControlsLayout/TableWithControlsLayout.scss

```diff
@@ -30,6 +30,8 @@
     &__table {
         position: relative;
         z-index: 2;
+
+        width: max-content;
```

Comment:
I am a liitle worried of this style possible impact on other tables

could you please clarify why this style was added